docs/pages/setup/reference/networking.mdx

```diff
@@ -5,9 +5,13 @@ description: Teleport ports and networking requirements
 
 ## Public address
 
-All three Teleport services (proxy, auth, node) have an optional
-`public_addr` property. The public address can take an IP or a DNS name. It can
-also be a list of values:
+<Tabs>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+
+All Teleport services (the Proxy Service, Auth Service, and Nodes) have an
```

Comment:
app service, db service, windows desktop service all have `public_addr` as well.

Reply:
At some point, I started conflating the term "Node" with "all Teleport services that aren't the Auth/Proxy." As you point out, this is incorrect, and our architecture overview guide [says as much](https://goteleport.com/docs/architecture/overview/#definitions). I still think we need some catch-all term for services that aren't the auth/proxy. I'll create a separate issue for this and fix my usage here.